src/api_authentication_app_skeleton/spec/requests/api/home_spec.cr

```diff
@@ -0,0 +1,11 @@
+require "../../spec_helper"
+
+describe "Home" do
+  it "says hello" do
+    request = AppRequest.new
+    
+    request.get("/")
+
+    request.response_json.should eq ({"hello" => "Hello World from Home::Index"})
+  end
+end
```

Comment:
This is missing a newline. You may want to run `crystal tool format src spec` to format everything